src/games/strategy/triplea/delegate/AirBattle.java

```diff
@@ -761,14 +762,8 @@ private void notifyCasualties(final GUID battleID, final IDelegateBridge bridge,
       public void run() {
         try {
           getRemote(firingPlayer, bridge).confirmEnemyCasualties(battleID, "Press space to continue", hitPlayer);
-        } catch (final ConnectionLostException cle) {
-          // somone else will deal with this
-          // System.out.println(cle.getMessage());
-          // cle.printStackTrace(System.out);
-        } catch (final GameOverException e) {
-          // ignore
         } catch (final Exception e) {
-          // ignore
+          ClientLogger.logQuietly(e);
```

Comment:
With that said, I don't think it would take too much to make a pretty big dent on that problem, and can add some more fixes in.

```
dan@dan-xps:~/work/triplea$ grep -r "ex.printStackTrace" | wc -l
80
```